CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to the "vscode-sops" extension will be documented in this fi
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
+## [Unreleased]
+### Added
+- Configuration `sops.enabled`, `sops.binPath` works now as expected.
+- Run control (config) file `.vscodesopsrc` for local project/workspace specific configuration (AWS profile & GCP credentials).
```

Comment:
Personally I would use something smaller like `.sopsrc` and claim it :) Let the sops devs also use it if they want!